### PR TITLE
Reformat asin #2045

### PR DESCRIPTION
**Pull request by vineel96**
Made changes to asin function, please review. issue: #2045

### CODE REVIEW

**Comment by RickSanchezStoic** [comment]
@vineel96 Kindly look through the changes that I have requested and let me know if you have any doubts, thanks!

**Review by RickSanchezStoic — CHANGES_REQUESTED** [review]

File: ivy/container/elementwise.py [hunk 1]
```diff
@@ -551,6 +561,16 @@ def asin(
...
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([0., 0.5, 1.]),\
+                              b=ivy.array([-4., 0.8, 2.]))
+        >>> y = x.asin()
+        >>> print(y)
+        {
+            a: ivy.array([0., 0.524, 1.57]),
+            b: ivy.array([nan, 0.927, nan])
+        }
```
[the remark this answers is not in the document]
Comment (vineel96) [reply ivy/container/elementwise.py:583]: @RickSanchezStoic  I have mage changes to all 3 requested changes, please check

**Comment by RickSanchezStoic** [comment]
@vineel96 please fix the `lint` errors. It the lambda function `asin` needs to be removed in mxnet.

**Comment by vineel96** [comment]
@RickSanchezStoic I have fixed the lint errors, please check

**Comment by RickSanchezStoic** [comment]
@vineel96 We shouldn't make any changes to any files or folders within `.idea`. Can you please fix that? Otherwise, everything looks good!

**Review by RickSanchezStoic — CHANGES_REQUESTED** [review]
Kindly make the requested change and then we can merge it.

**Comment by vineel96** [comment]
@RickSanchezStoic I have fixed the issue, please check

**Comment by RickSanchezStoic** [comment]
Looks good!

**Pull request merged**